wled00/audio_source.h

```diff
@@ -9,7 +9,7 @@
 */
 
 // data type requested from the I2S driver - currently we always use 32bit
-//#define I2S_USE_16BIT_SAMPLES   // (experimental) define this to request 16bit - more efficient but possibly less compatible
+#define I2S_USE_16BIT_SAMPLES   // (experimental) define this to request 16bit - more efficient but possibly less compatible
```

Comment:
This change degrades all I2S mics to 16 bit.
Please undo the change, and it the `build_flags` of the board - in case you think its absolutely necessary.